interactors/advanced-search.js

```diff
@@ -0,0 +1,33 @@
+import Button from './button';
+import Select from './select';
+import TextField from './text-field';
+import HTML from './baseHTML';
+
+export const AdvancedSearchRow = HTML.extend('advanced search row')
+  .selector('[class*=AdvancedSearchRow-]')
```

Comment:
can it be more special, something which begins from ^ . Or added hardcoded attribute(id, name, index for example)

Reply:
I'm not sure what else we can use here... there are 6 rows on the page so we can't use ids.
But you can search a row by index! For example `AdvancedSearchInteractor({ index: 0 })`